src/localization/i18n.js

```diff
@@ -61,4 +61,12 @@ i18n.services.formatter.add('lowercase', (value, lng, options) => {
   return i18n.t(value).toLowerCase();
 });
 
+i18n.services.formatter.add('andList', (value, lng, options) => {
```

Comment:
question: can we use [i18n's built-in list support?](https://www.i18next.com/translation-function/formatting#list) 
```
// JSON
{
  "intlList": "A list of {{val, list}}"
}

i18next.t('intlList', { val: ['locize', 'i18next', 'awesomeness'] });
// --> A list of locize, i18next, and awesomeness
```